__tests__/app.test.js

```diff
@@ -156,6 +157,61 @@ describe('GET /api/articles', () =>
             expect(body).toBeSortedBy('created_at', {ascending: true})
         })
     })
+    test('GET will respond with an array of objects only with the relevant topic if specified in the query', () =>
+    {
+        return request(app).get('/api/articles?topic=mitch').expect(200)
+        .then(({ body }) =>
```

Comment:
We want another test where we check for a status 200, for when the topic query is valid but no arrays exist -> should respond with an empty array